dev/packages/example/endpoint-0.0.1/manifest.yml

```diff
@@ -11,6 +11,19 @@ release: beta
 type: solution
 license: basic
 
+
+
+datasources:
+  -
+    name: endpoint
+    title: Endpoint data source
+    description: Interact with the endpoint.
+
+    # This tells the UI that for configuration, it must link to a specific solution
+    # Only solution packages can contain this field.
+    solution: endpoint
```

Comment:
@jen-huang Will this help to link to endpoint when someone clicks and endpoint datasource? Or selects endpoint as a datasource?

Reply:
++ I like this field. We can treat the value as the app in Kibana to link to (`/app/{solution}`), or hardcode a solution<->app url mapping in the code. I'm not sure if this is enough for what to display in Step 2 of the data source wizard though. It will currently display "Nothing to configure" and if the user confirms the addition, it will add a datasource with an empty `inputs` array

Reply:
I like the name for that, good idea to add a dedicated field for it.

Reply:
it could be also a function call or a handler that a solution register to us, we can then pass the content of the datasource for disambiguation, this is where packages or other fields could be used to display the appropriate UI.

Reply:
@jen-huang I think the user should never end up on page 2 of the wizard but be redirected to endpoint for step 2.